src/Element/Select2.php

```diff
@@ -152,8 +152,9 @@ public static function processSelect(&$element, FormStateInterface $form_state,
       }
     }
 
-    if (!$element['#multiple'] && !isset($element['#options'][''])) {
-      $empty_option = ['' => ''];
+    $empty_value = isset($element['#empty_value']) ? $element['#empty_value'] : '';
```

Comment:
I think we could do this here:
```suggestion
    $empty_value = $element['#empty_value'] ?? '';
```

Reply:
`??` supported since PHP 7 only
but yeah, it's time to forget PHP 5 completely :)